mne/io/eeglab/tests/test_eeglab.py

```diff
@@ -0,0 +1,18 @@
+# Author: Mainak Jas <mainak.jas@telecom-paristech.fr>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+
+from mne.io import read_raw_eeglab
+from mne.io.tests.test_raw import _test_raw_reader
+
+base_dir = op.join(op.dirname(op.realpath(__file__)), 'data')
+fname = op.join(base_dir, 'eeglab_data.set')
+montage = op.join(base_dir, 'eeglab_chan32.locs')
+
+
+def test_io_set():
+    """Test importing EEGLAB .set files"""
+    _test_raw_reader(read_raw_eeglab, True, input_fname=fname,
```

Comment:
I think this option is gone due to your commit yesterday: https://github.com/mne-tools/mne-python/commit/4552785a95ebbe86c5e9ab8f5be5e5d7ef96bcc2